BaragonService/src/main/java/com/hubspot/baragon/service/worker/BaragonRequestWorker.java

```diff
@@ -190,6 +189,8 @@ public void run() {
           }
         }
       }
+
+      requestManager.failStuckPendingRequests();
```

Comment:
i'm flip-flopping on whether or not we want this check to occur every time the request worker is triggered -- could be expensive for people who haven't enabled purging yet. Could we move this into one of the other background workers? Or investigate doing all the request creation stuff as part of a ZK transaction?